src/components/pager/pager.component.js

```diff
@@ -104,12 +104,18 @@ const Pager = ({
 
   const sizeSelector = () => {
     return (
-      <StyledDropdown
-        options={ Immutable.fromJS(pageSizeSelectionOptions) }
+      <StyledSelect
         value={ currentPageSize }
         onChange={ (ev) => { handleOnPagination(ev); } }
         data-element='page-select'
-      />
+      >
+        { pageSizeSelectionOptions.map(sizeOptions => (
```

Comment:
Very minor but if I'm reading this right, this might make more sense as `sizeOption` rather than `sizeOptions`